content/docs/quickstart.mdx

```diff
@@ -22,74 +22,216 @@ keywords:
+
+![Viewing the two starter routes](./img/quickstart/pz-quickstart-routes.png)
+
+### Connect to SSH
+
+In a terminal, enter the following command and replace `CLUSTER_STARTER_DOMAIN` with your own:
 
 ```bash
-docker compose up
+pomerium-cli tcp ssh.<CLUSTER_STARTER_DOMAIN>.app:22
+```
+
+A successful request results in a response similar to the one below:
+
+```shell-session
+$ pomerium-cli tcp ssh.dominant-cowbird-8176.pomerium.app:22
+2024/03/25 16:18:45 listening on 127.0.0.1:52541
 ```
 
-Access the **verify route** you built in your policy: `https://verify.localhost.pomerium.io`
+:::note
 
-If you get a self-signed certificate warning, see [Handle Self-Signed Certificate Warning](/docs/internals/troubleshooting#handle-self-signed-certificate-warning) to bypass it.
+If you haven't, install [**Pomerium CLI**](/docs/deploy/clients/pomerium-cli) with any of the following methods:
 
-You should be redirected to the **verify** service. You'll see a page like this:
+<Tabs>
+<TabItem value="docker" label="Docker">
 
-![Verify page](./img/quickstart/verify-service.png)
+```bash
+docker run cr.pomerium.com/pomerium/cli:latest
+```
 
-Although identity verification failed, you successfully integrated Pomerium with the upstream verify service.
+</TabItem>
+<TabItem value="yum" label="Yum">
+
+```bash
+[pomerium-pomerium]
+name=pomerium-pomerium
+baseurl=https://dl.cloudsmith.io/public/pomerium/pomerium/rpm/el/$releasever/$basearch
+repo_gpgcheck=1
+enabled=1
+gpgkey=https://dl.cloudsmith.io/public/pomerium/pomerium/gpg.6E388440B94E1407.key
+gpgcheck=1
+sslverify=1
+pkg_gpgcheck=1
+```
 
-:::tip
+</TabItem>
+<TabItem value="deb" label="Deb">
 
-Because this guide doesn't include a [signing key](/docs/reference/signing-key) in the configuration, identity verification will fail.
+```bash
+curl -1sLf 'https://dl.cloudsmith.io/public/pomerium/pomerium/gpg.6E388440B94E1407.key' | apt-key add -
+echo "deb https://dl.cloudsmith.io/public/pomerium/pomerium/deb/debian buster main" > /etc/apt/sources.list.d/pomerium-pomerium.list
+```
+
+</TabItem>
+<TabItem value="binaries" label="Binaries">
+
+```bash
+ARCH=[your arch]
+OS=[your os]
+VERSION=[desired version]
+curl -L https://github.com/pomerium/cli/releases/download/${VERSION}/pomerium-cli-${OS}-${ARCH}.tar.gz \
+    | tar -z -x
+```
 
-See [Identity Verification](/docs/capabilities/getting-users-identity) for more information on how Pomerium can use JWTs for authentication.
+</TabItem>
+<TabItem value="homebrew" label="Homebrew">
+
+```bash
+brew tap pomerium/tap
+brew install pomerium-cli
+```
+
+</TabItem>
+<TabItem value="source" label="Source">
+
+```bash
+git clone git@github.com:pomerium/cli.git
+cd pomerium
+make build
+./bin/pomerium-cli --help
+```
+
+</TabItem>
+
+</Tabs>
 
 :::
 
-## Next Steps
+### Connect to Verify
+
+In the Zero Console, select the **From** URL for the **Verify** app.
```

Comment:
Either above or here, I think this would definitely benefit from some explanation — what is the verify service? what does it demonstrate?

Reply:
I think it makes most sense to include it at the first mention of the Verify app, so earlier in the page. Will do.